posts/2020/11/Design In Open Source

```diff
@@ -0,0 +1,187 @@
+are (possibly) documented only in their mind). If members of a project don’t intentionally take time  
+to prioritize design, it can be easier to just merge changes directly into the code base and see how  
+the community likes it, as opposed to putting significant amounts of time into design. Many features  
+that end up in open-source software are the result of one or two software developers writing the code  
+and offering it directly to the project, while even the most well executed design still needs to be  
+coded. It often happens that the solution that’s working now, is the priority over the solution that  
+might work better, *if* it is even built.
+**IPF:** Because features are often already or mostly built by the time they are revealed to the team  
+(designers included), they have also already been designed—intentionally or not—by the people who built  
+that feature. This process works against involving designer perspectives where they might be most  
+helpful or impactful: when initially solving the problem.
+
+## To a brighter future
+  
+**TG:** In my experience, which is mostly in Jupyter-related projects, design input is desired, and well  
+received. Different projects, depending on size and scope, have different understandings of, and  
+interest in, design. If designers are going to make a larger impact on open-source projects, committers  
+need to understand the value of design beyond simply ‘making things look good.’ Many open-source  
+initiatives would benefit from a visual overhaul, especially of the type that leans into consistent  
+looks for similar interactions throughout the software. This is usually the limits of other contributors'  
+expectations of designers, but that’s just scratching the surface of the problems we’ve worked through  
+so far, and what we’re capable of in the future.
+
+**IPF:** At the beginning, we mentioned imagining a better world as a part of being a designer, and  
+imagining a healthier open source experience is no exception! Changing the collaboration process is a  
+good first step.
+
+**TG:** As a Human-Computer Interaction practitioner, I always strive to understand ‘why’ a problem exists  
+before I decide what to do about it. There’s a lot of ‘what’ and ‘how’ in open-source development.  
+Oftentimes, a scientist or developer simply solves the problem they have. But if we take some time to  
+walk through the design process and understand more about why the question arose and who else has the problem,  
+we can get to the core issue, and design both a solution and interface for it in a way that is knowledgeable  
+both of the problem and the people trying to solve it. In essence, we build software that the user can partner  
+with to process data, instead of building a tool that accomplishes a single task.
+ 
+New -- **IPF:** But if designers are brought in just at the review or UI decision stage, as they often are, it  
```

Comment:
I think the `New --` here can be dropped - is that right?

Reply:
yes!